package/yast2-registration.changes

```diff
@@ -1,3 +1,8 @@
+-------------------------------------------------------------------
+Thu Dec 14 08:45:40 UTC 2017 - mfilka@suse.com
+
+- added support for reading registration code from install.inf
```

Comment:
ETOOFEWDIGITS, can we add [FATE#323450](https://fate.suse.com/323450) "Allow installation of different products with one installation medium" to this?
Also, is it OK not to bump the version?

Reply:
well it was originally developed due to internal requirements, but I will add both.